README.md

```diff
@@ -30,6 +30,10 @@ Sway is an incredible window manager, and certainly one of the most well establi
     - `blur_xray enable|disable`: this will set floating windows to blur based on the background, not the windows below. You probably want to set this to `disable` :)
     - `blur_passes <integer value 0 - 10>`
     - `blur_radius <integer value 0 - 10>`
+    - `blur_noise <float percentage 0 - 1>`
+    - `blur_brightness <float percentage 0 - 2>`
+    - `blur_contrast <float percentage 0 - 2>`
+    - `blur_saturation <float percentage 0 - 2>`
```

Comment:
Sorry to be so nitpicky, let's keep this as `<float value`, in case users think they need to add the `%` sign or something.